proposals/lip-0053.md

```diff
@@ -717,7 +717,7 @@ def execute(trs: Transaction) -> None:
             # In this case, we do not even update the chain account with the new certificate.
             return
         try:
-            # The CCM must come be directed to the sidechain, unless it was bounced on the mainchain.
+            # The CCM must be directed to the sidechain, unless it was bounced on the mainchain.
```

Comment:
this will be over-written with the refactoring PR and may cause merge conflicts.

Reply:
This should be fixed by the typo fixing PR